dWorldServer/WorldServer.cpp

```diff
@@ -234,6 +235,33 @@ int main(int argc, char** argv) {
 		Game::physicsWorld = &dpWorld::Instance(); //just in case some old code references it
 		dZoneManager::Instance()->Initialize(LWOZONEID(zoneID, instanceID, cloneID));
 		g_CloneID = cloneID;
+
+		// pre calculate the FDB checksum
+		if (bool(std::stoi(Game::config->GetValue("check_fdb")))) {
```

Comment:
This will fail if there is no config value in the ini file, destroying backward comparability. Just compare the raw string with "1".